tests/controllers/excludename/matchAll/chainsaw-test.yaml

```diff
@@ -0,0 +1,131 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Authors of Nimbus
+
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: multiple-ns-add-csib
+spec:
+  description: >
+    1. Create prod, dev, staging ns
+    2. Add security intent
+    3. Add csib 
+    4. Delete prod namespace
+    5. Create prod namespace again
```

Comment:
The description should explain the purpose of the test and what it involves.  Currently, it seems like you've included the steps involved in performing the test instead of a general overview.

For e.g,.
https://github.com/5GSEC/nimbus/blob/5a4217460ea4adb01a0a1afab9e4634b7b924e1a/tests/controllers/securityintentbinding/create/chainsaw-test.yaml#L9-L11

Reply:
Will be adding a meaningful description in a separate PR